qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -141,17 +141,21 @@ def _init_stylesheet(profile):
 
     css = shared.get_user_stylesheet()
     source = """
-        (function() {{
+        (function setStylesheet() {{
+            if (document.head === null) {{
+                setTimeout(setStylesheet);
```

Comment:
I know very little about js, so I could be completely wrong, but shouldn't there be a timeout in ms here? Wouldn't this cause an infinite loop in js until the page is loaded? If this is normal convention then please ignore me though :).

I guess if we put a wait time here, it might cause a flicker of that time, though.

Reply:
An omitted delay has the same effect as specifying 0 ms, but the function is still not executed immediately. I was also concerned that it may impact performance by being executed repeatedly. When I tested this, however, `setTimeout()` was always called exactly once, so I don't believe that this will be a problem.